### ISSUE TITLE
Photo collection Permissions

### ISSUE THREAD

**Issue by thob** [MobiusSync/MobiusSync#182]
It seems, after syncing, I only have read-only permissions on the PC (on Linux) in file level. Anything I can do to change this? I'm currently sharing the Photo  collection with an always-on PC and my notebook.

permissions for dirs
![2024-07-23_17-40](https://github.com/user-attachments/assets/942fb511-8a51-4a12-8461-a8695e64a277)

permissions for files
![2024-07-23_17-41](https://github.com/user-attachments/assets/c1c7dfc4-9cb8-4cdb-93f2-81bd6526e73b)

thanks

**Comment by SimonPickup** [comment]
This was a design choice to reflect that it's one-way sync, i.e. to prompt you to think before you modify/delete files.

Maybe it could be configurable?

**Comment by thob** [comment]
I would prefer to be able easily delete, so a checkbox to configure it would be fantastic!

**Comment by thob** [comment]
It seems I’m actually after bi-directional sync. I just realized I can’t delete the files on the iPhone

**Comment by SimonPickup** [comment]
OK, sync back of deletions is covered in #173.

Is anyone interested in having the exported permissions be configurable then? Please upvote or comment here.

**Comment by thob** [comment]
Absolutely, I would be happy if I can easily deal with snyced pics

**Comment by rotbaer-peter** [comment]
I agree - it‘s the same with MacOS and I‘m voting for configurable exportet permissions!

**Comment by SimonPickup** [comment]
Added a new Photo collection option "Export as red-only" which is on by default.

**Comment by SimonPickup** [comment]
Reopening as I had closed prematurely. This is in TestFlight, not public release.
